src/Admin/Controllers/UsersController.cs

```diff
@@ -63,20 +56,11 @@ public async Task<IActionResult> Index(string email, int page = 1, int count = 2
 
         var skip = (page - 1) * count;
         var users = await _userRepository.SearchAsync(email, skip, count);
-
-        if (_featureService.IsEnabled(FeatureFlagKeys.MembersTwoFAQueryOptimization))
```

Comment:
The feature flag lookup, should be kept here. We only want to execute that query if its enabled